tt_stl/CMakeLists.txt

```
@@ -0,0 +1,15 @@
+add_library(tt_stl INTERFACE)
+add_library(Metalium::Metal::STL ALIAS tt_stl)
```

It's not part of Metalium anymore.

```suggestion
add_library(TT::STL ALIAS tt_stl)
```